docs/source/md/software_design_overview.md

```diff
@@ -0,0 +1,110 @@
+# Software Development Process
+
+This document provides a high-level overview of the software development process that our team uses for their
+repositories (most importantly [InnerEye-DeepLearning](https://github.com/microsoft/InnerEye-DeepLearning) and
+[InnerEye-Gateway](https://github.com/microsoft/InnerEye-Gateway)).
+
+For detailed guidance for developers, please refer to the [coding guidelines](coding_guidelines.md).
```

Comment:
This is not a valid link in this repository.

Reply:
Scrap above, just seen your e-mail to Tom. For full decoupling from hi-ml the coding guidelines will also need to be copied over!

Reply:
I removed the link altogether. We have other docs in InnerEye that very much overlap with the coding guidelines.